office/libreoffice/Portfile

```diff
@@ -0,0 +1,328 @@
+                    size    320952 \
+                    mdds-1.6.0.tar.bz2 \
+                    rmd160  003175ca1b3d1293a6f144490de94eba72cb1697 \
+                    sha256  f1585c9cbd12f83a6d43d395ac1ab6a9d9d5d77f062c7b5f704e24ed72dae07d \
+                    size    350406 \
+                    openssl-1.0.2t.tar.gz \
+                    rmd160  60fa7238a3beefb1e95d76de607d69af7198118b \
+                    sha256  14cb464efe7ac6b54799b34456bd69558a749a4931ecfd9cf9f71d7881cac7bc \
+                    size    5355422 \
+                    xmlsec1-1.2.30.tar.gz \
+                    rmd160  5de6607d95b852da826f99ca6ca89a81f71aaaf3 \
+                    sha256  2d84360b03042178def1d9ff538acacaed2b3a27411db7b2874f1612ed71abc8 \
+                    size    1989670 \
+                    884ed41809687c3e168fc7c19b16585149ff058eca79acbf3ee784f6630704cc-opens___.ttf \
+                    rmd160  00b1e296adf2caad44c49ad9743e5da7c771e480 \
+                    sha256  884ed41809687c3e168fc7c19b16585149ff058eca79acbf3ee784f6630704cc \
+                    size    207544 \
+                    Python-3.7.7.tar.xz \
+                    rmd160  b36217d3ee4566d69e02076c8ff7bfa0de387b4f \
+                    sha256  06a0a9f1bf0d8cd1e4121194d666c4e28ddae4dd54346de6c343206599f02136 \
+                    size    17268888
+
+depends_build-append \
+    port:autoconf \
+    port:automake \
+    port:bison \
+    port:coreutils \
+    port:cppunit \
+    port:flex \
+    port:gettext \
+    port:gperf \
+    port:intltool \
+    port:pkgconfig \
+    port:unixODBC
+depends_lib-append  \
+    port:apr \
+    port:apr-util \
+    port:boost \
+    port:bzip2 \
+    port:curl \
+    port:expat \
+    port:gdbm \
+    port:gettext \
+    port:graphite2 \
+    port:harfbuzz \
+    port:harfbuzz-icu \
+    port:hunspell \
+    port:hyphen \
+    port:icu \
+    port:jpeg \
+    port:lcms2 \
+    port:libcdr-0.1 \
+    port:libepoxy \
+    port:libetonyek \
+    port:libfreehand \
+    port:libiconv \
+    port:liblangtag \
+    port:libmspub \
+    port:libodfgen \
+    port:libpagemaker \
+    port:libpng \
+    port:librevenge \
+    port:libvisio-0.1 \
+    port:libwpd-0.10 \
+    port:libwpg-0.3 \
+    port:libwps \
+    port:libxml2 \
+    port:libxslt \
+    port:ncurses \
+    port:nspr \
+    port:nss \
+    port:openssl \
+    port:ossp-uuid \
+    port:raptor2 \
+    port:readline \
+    port:redland \
+    port:serf1 \
+    port:sqlite3 \
+    port:xz \
+    port:zlib
+
+if {${os.major} < 14} {
+    pre-fetch {
+        ui_error "${name} @${version} requires OS X 10.10 or later."
+        return -code error "incompatible OS X version"
+    }
+}
+
+patch {
+    # Force GNU find.
+    reinplace "s|find |gfind |g" ${worksrcpath}/bin/unpack-sources
+    # Undefined reference.
+    reinplace "221d" ${worksrcpath}/vcl/source/filter/ipdf/pdfread.cxx
+}
+
+set product_name LibreOffice
+
+# This needs the full Xcode SDK. CLT is not enough.
```

Comment:
This comment can be deleted since it tells us nothing more than what the code already says.
```suggestion
```